sympy/series/tests/test_limits.py

```diff
@@ -276,6 +276,12 @@ def test_issue_5164():
     assert limit(x**(-0.5), x, 4) == S(4)**(-0.5)
 
 
+def test_issue_14793():
+    expr = ((x + S(1)/2) * log(x) - x + log(2*pi)/2 - \
+        log(factorial(x)) + S(1)/(12*x))*x**3
+    assert simplify(limit(expr, x, oo) - 1/360) == 0
```

Comment:
This should be `assert limit(expr, x, oo) == S(1)/360`.

Reply:
I see. Are there any particular disadvantage of the way done previously?

Reply:
The call to `simplify()` function is always costly, it should be avoided as much as possible.

Reply:
Okay, I will make the required changes.